### DISCUSSION TITLE
GTA⁵ ON MICEWINE

### DISCUSSION THREAD

**Discussion by WINDROID-EMU** [ptitSeb/box64#1828] [General]
![GTA⁵ LOG.txt](https://github.com/user-attachments/files/17007588/GTA.LOG.txt) Hi Sebastian, this is the GTA 5 log. It opens and loads for about 5 seconds and then closes. I'm on micewine

**Comment by ptitSeb** [comment]
There is no box eror on this log, only wine ones..

Also, note that GTA5 needs a patch on Wine to be able to use keyboard properly (all the keyboard messages)... that's a wine known issue (a patch exist on proton).

**Reply by WINDROID-EMU** [reply]
thank you sebastian sorry to bother you
